lib/keys.js

```diff
@@ -41,7 +41,20 @@ Keys.prototype.create = function(fn) {
 
     data[key] = true;
     keys.writeFile(data, function(err) {
-      fn(err, key);
+      if(err){
+        if(err.code == 'ENOENT' && err.path == '.dpd/keys.json'){
+          console.error('Path: '+err.path+' not found, creating new one.');
+          // TODO: Could ask with prompt?
+          fs.mkdirSync('.dpd');
+          keys.writeFile(data,function(error){
```

Comment:
I could implement that. But I dont like whole async/callback implementation of it, so i would implement it in sync methods. I think it would kill it because its just a cmd code. It doesnt need to be async. Right?